lnst/RecipeCommon/Perf/Evaluators/BaselineTcRunAverageEvaluator.py

```diff
@@ -29,12 +29,12 @@ def compare_result_with_baseline(
                 )
             ]
         elif (threshold := self._thresholds.get(metric_name, None)) is not None:
-            difference = ((result.time_taken / baseline.time_taken) * 100 ) - 100
+            difference = ((result.rule_install_rate.average / baseline.rule_install_rate.average) * 100) - 100
             direction = "higher" if difference >= 0 else "lower"
             text = [
                 f"{self.__class__.__name__} of tc run with {metric_name}",
                 f"{result.description}",
-                f"baseline time_taken={baseline.time_taken}",
+                f"baseline rules_per_sec={baseline.rule_install_rate.average}",
```

Comment:
checked the newer job you sent me and i think this could be a bit more descriptive. what you have now:
```
        host1.ens5f0np0 multi tc run num_instances=4 mean time_taken=10.508329783249991s num_rules=(100000, 100000, 100000, 100000)
        baseline rules_per_sec=9552.86526092749
        -0.383185 lower than baseline 
        Allowed differences: 1.42% 
```

so you're comparing "time taken" which is printed in the `result.description` part, but comparing to `rules rate` printed here... i think it would be a bit nicer to see `rules rate` to `rules rate` comparison here.

Reply:
Yeah this description could probably be clean up a bit.

Reply:
Fixed let me know what you think heres what it looks like now:
```
    host1.ens1f0np0 multi tc run num_instances=1 mean time_taken=3.740612900000997s num_rules=(100000,)
    Baseline: 27106.8640311474 rules/sec
    Measured: 26733.586894268945 rules/sec
    1.377058% lower than baseline 
    Allowed difference: 3.0% 
```

Reply:
looks good